source/common/config/subscription_factory_impl.cc

```diff
@@ -86,8 +86,12 @@ SubscriptionPtr SubscriptionFactoryImpl::subscriptionFromConfigSource(
     }
   }
   case envoy::config::core::v3::ConfigSource::ConfigSourceSpecifierCase::kAds: {
+    Config::GrpcMuxSharedPtr gmuxsp = cm_.adsMux();
```

Comment:
probably an `auto` here would suffice? Or skip the local var altogether, not sure if it improves readability.